### DISCUSSION TITLE
Github action build fails at "Move jars"

### DISCUSSION THREAD

**Discussion by elect86** [community/community#25440] [Actions]
<p>As titled, <a href="https://github.com/kotlin-graphics/imgui/runs/1264651020?check_suite_focus=true" rel="noopener nofollow ugc">here</a> the error</p>
<pre><code class="lang-auto">  Run mkdir shadowJars &amp;&amp; mv */build/libs/imgui-*.jar shadowJars
  mkdir shadowJars &amp;&amp; mv */build/libs/imgui-*.jar shadowJars
  shell: /bin/bash -e {0}
  env:
    JAVA_HOME_11.0.8_x64: /opt/hostedtoolcache/jdk/11.0.8/x64
    JAVA_HOME: /opt/hostedtoolcache/jdk/11.0.8/x64
    JAVA_HOME_11_0_8_X64: /opt/hostedtoolcache/jdk/11.0.8/x64
mv: cannot stat '*/build/libs/imgui-*.jar': No such file or directory
Error: Process completed with exit code 1.
</code></pre>
<p>This is the folder content:</p>
<blockquote>
<p>imgui-html-doc.jar<br>
imgui-javadoc.jar<br>
imgui-sources.jar<br>
imgui.jar</p>
</blockquote>
<p>Either if it should not fail because there are actually matches for <code>imgui-*.jar</code>, I have no idea where this is specified.</p>
<p>Also, how can I fix this?</p>
<p>Thanks in advance</p>

**Comment by airtower-luna** [comment]
<aside class="quote no-group" data-username="elect86" data-post="1" data-topic="138672">
<div class="title">
<div class="quote-controls"></div>
 elect86:</div>
<blockquote>
<p>This is the folder content:</p>
</blockquote>
</aside>
<p>The content of which folder? The first asterisk in <code>*/build/libs/imgui-*.jar</code> might match many folders, or none, which might be the problem here.</p>

**Comment by elect86** [comment]
<aside class="quote no-group" data-username="airtower-luna" data-post="2" data-topic="138672" data-full="true">
<div class="title">
<div class="quote-controls"></div>
 airtower-luna:</div>
<blockquote>
<p>The content of which folder? The first asterisk in <code>*/build/libs/imgui-*.jar</code> might match many folders, or none, which might be the problem here.</p>
</blockquote>
</aside>
<p>The folder where it does look for: <code>/build/libs/</code></p>

**Comment by airtower-luna** [comment, marked as answer]
<p>Then that’s indeed the problem. <code>*/build/libs/imgui-*.ja</code> would only match if your JARs were in directories with another leading directory, e.g. <code>abc/build/libs/</code> or <code>xyz/build/libs/</code>. Try:</p>
<pre><code class="lang-auto">mv build/libs/imgui-*.jar shadowJars/
</code></pre>

**Comment by elect86** [comment]
<p>Uh, I got it now what you meant… thanks dude!</p>

**Comment by airtower-luna** [comment]
<p>You’re welcome, but there’s no dude here. :wink:</p>

**Comment by elect86** [comment]
<p>Dudine then :slight_smile:</p>
<p>(I just discovered it, I didn’t know that)</p>
